foo_067_rw0_crop_suitability_class/foo_067_rw0_crop_suitability_class_processing.py

```diff
@@ -0,0 +1,98 @@
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/ENSEMBLE/rcp8p5/2050sH/scHr_cot.tif',
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/ENSEMBLE/rcp4p5/2050sH/scHr_cot.tif',
+        # coffee tifs
+            # irrigation
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/CRUTS32/Hist/8110H/scHi_cof.tif',
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/ENSEMBLE/rcp4p5/2020sH/scHi_cof.tif',
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/ENSEMBLE/rcp8p5/2020sH/scHi_cof.tif',
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/ENSEMBLE/rcp4p5/2050sH/scHi_cof.tif',
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/ENSEMBLE/rcp8p5/2050sH/scHi_cof.tif',
+            # rainfed
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/CRUTS32/Hist/8110H/scHr_cof.tif',
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/ENSEMBLE/rcp4p5/2020sH/scHr_cof.tif',
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/ENSEMBLE/rcp8p5/2020sH/scHr_cof.tif',
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/ENSEMBLE/rcp4p5/2050sH/scHr_cof.tif',
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/ENSEMBLE/rcp8p5/2050sH/scHr_cof.tif',
+        # rice tifs
+            # NOTE - only have historical tifs for wetland/dryland rice - contacting GAEZ team to see if they can
+            # deliver ENSEMBLE tifs for future time periods.
+            # wetland rice - gravity irrigation
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/CRUTS32/Hist/8110H/scHg_rcw.tif',
+            # wetland rice - rainfed
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/CRUTS32/Hist/8110H/scHr_rcw.tif',
+            # dryland rice - rainfed
+            'https://s3.eu-west-1.amazonaws.com/data.gaezdev.aws.fao.org/res05/CRUTS32/Hist/8110H/scHr_rcd.tif'
+        ]
+
+# download tifs and rename as they are downloaded because some have the same name when downloaded
+# files are renamed based on their unique path information that is contained in the urls
+raw_data_files = []
+for url in url_list:
+    # split URL to access path info
+    s = urlsplit(url)
+    # swap "/" for "-" in the path
+    r = re.sub("/", "-", s.path)
+    # remove the beginning portion of path which is common to all urls
+    p = r.replace("-data.gaezdev.aws.fao.org-res05-", "")
+    # create a new path and filename
+    filename = os.path.join(data_dir, p)
+    # download data and save with new filename in data_dir
+    d = urllib.request.urlretrieve(url, filename)
+    raw_data_files.append(d)
+
+'''
+Process data
```

Comment:
@alxswny can you add `processed_data_file`

Reply:
@chrowe added -- is that all that's needed?